### PR TITLE
Refute bad names when creating

### PR DESCRIPTION
**Pull request by ramn**

### CODE REVIEW

**Comment by Keats** [comment]
How about turning names with space into slugs ?

**Comment by ramn** [comment]
Yes that could be an improvement but meanwhile I think it would be good that the validation logic that is run when a migration is executed, is also run when the migration is created.

**Comment by ramn** [comment]
Suggestion for slugification

File: src/files.rs [hunk 1]
```diff
@@ -139,9 +142,16 @@ fn parse_filename(filename: &str) -> MigrateResult<MigrationFile> {
     Ok(MigrationFile::new(filename, name, number, direction))
 }
 
+fn slugify(s: &str) -> String {
```
Comment (Keats) [inline src/files.rs]: this method does too much and is not really a slug (slug use `-`, not `_`).

How about `s.replace(" ", "-")` and letting the parse_filename handle invalid names?

**Comment by ramn** [comment]
What do you think?

**Comment by Keats** [comment]
I think that's ok, i'll probably change a couple of things before releasing though

**Comment by Keats** [comment]
I pushed something for that (very basic, people can edit filenames themselves if they want to) with error handling, so closing this.

Will release a new version in a few minutes

**Comment by ramn** [comment]
Nice, thx!

**Pull request closed without merging**